overlord/snapstate/backend_test.go

```diff
@@ -172,13 +172,18 @@ type snapSpec struct {
 	Name     string
 	Channel  string
 	Revision snap.Revision
+	cohort   string
```

Comment:
any particular reasons this is unexported? (is test code so it doesn't matter as much) the rest of the fields is exported